lib/shape.ml

```diff
@@ -41,3 +72,19 @@ let init () =
 let close () =
   ignore (read_line ());
   close_graph ()
+
+  let () =
+  init ();
+  set_color black;
+
+  (* Create shapes *)
+  let r1 = rectangle 200 100 in
+  let r2 = translate 100 0 r1 in
+  let c = circle 100 in
+  let e = ellipse 40 20 in
+  let t = triangle ~x1:50 ~y1:50 ~x2:100 ~y2:150 ~x3:150 ~y3:50 () in
+
+  (* Display shapes *)
+  show [r1; r2; c; e; t];
+
+  close ()
```

Comment:
`shape.ml` provides the interface people can use to create shapes. As such, if you want to test your new features added to shape, this is not the right place -- it needs to go to `examples/` or `tests/`. A thing to remember would be that `shape.ml` shouldn't contain anything that's automatically executed, since it just provides ways for users to build the shapes they want. I think this can go as we have similar tests and examples.